Source/Evolve.cpp

```diff
@@ -149,7 +156,48 @@ void interpolate_rhs_from_mesh(FlavoredNeutrinoContainer& neutrinos_rhs, const M
     const int shape_factor_order_x = geom.Domain().length(0) > 1 ? SHAPE_FACTOR_ORDER : 0;
     const int shape_factor_order_y = geom.Domain().length(1) > 1 ? SHAPE_FACTOR_ORDER : 0;
     const int shape_factor_order_z = geom.Domain().length(2) > 1 ? SHAPE_FACTOR_ORDER : 0;
-
+    
+    //Create EoS table object
+    using namespace nuc_eos_private;
+    EOS_tabulated EOS_tabulated_obj(alltables, epstable, logrho, logtemp, 
+                                    yes, helperVarsReal, helperVarsInt);
+
+    //Create NuLib table object
+    using namespace nulib_private;
+    NuLib_tabulated NuLib_tabulated_obj(alltables_nulib, logrho_nulib, logtemp_nulib, 
+                                        yes_nulib, helperVarsReal_nulib, helperVarsInt_nulib);
+
+    
+    
+    //---------------------------
+    const int lev = 0;
+   
+#ifdef _OPENMP
+#pragma omp parallel
+#endif
+    for (FNParIter pti(neutrinos_rhs, lev); pti.isValid(); ++pti)
+    {
+        const int np  = pti.numParticles();
+        //printf("(Evolve.cpp) number of particles: np = %d \n", np);
+        FlavoredNeutrinoContainer::ParticleType* pstruct = &(pti.GetArrayOfStructs()[0]);
+
+        amrex::ParallelFor (np, [=] AMREX_GPU_DEVICE (int i) {
```

Comment:
I have cleaned this loop and commented it out. But I would like to keep the commented code since it can be used for printing values for each particle in case particle debug is needed in the future.